From 5c8441e17b21f6b7cc9da641fff0a118a22df780 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Mon, 20 Mar 2017 09:12:50 +0100
Subject: [PATCH 1/6] update changelog
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/changelog | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index e7ab154..52ec3a9 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -4,6 +4,30 @@ ceph (12.0.1-1) stable; urgency=medium
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Fri, 24 Mar 2017 15:47:57 +0000
 
+ceph (12.0.0-pve3) stable; urgency=medium
+
+  * make ceph version output even more explicit
+
+  * enable systemd targets by default
+
+  * fix IPv6 monitor creation bug
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 22 Mar 2017 09:02:37 +0100
+
+ceph (12.0.0-pve2) stable; urgency=medium
+
+  * support Debian Stretch in ceph-detect-init
+
+  * fix "ceph version" output
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 21 Mar 2017 09:58:27 +0100
+
+ceph (12.0.0-pve1) stable; urgency=medium
+
+  * Rebuild for PVE 5.0 / Debian Stretch
+
+ -- Proxmox Support Team <support@proxmox.com>  Mon, 20 Mar 2017 09:12:32 +0100
+
 ceph (12.0.0-1) stable; urgency=medium
 
   * New upstream release
-- 
2.1.4

