From d98627902c7534966e57c1efaf0d251556af751e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Mon, 20 Mar 2017 09:12:50 +0100
Subject: [PATCH 1/7] update changelog
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/changelog | 52 ++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 52 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 147e186..b394148 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,21 +1,73 @@
+ceph (12.0.3-pve1) stable; urgency=medium
+
+  * rebase on top of v12.0.3
+
+  * don't enable (broken) ceph-mgr module by default
+
+  * use bundled boost
+
+  * fix Ceph #15569: two logrotate scripts when upgrading from Debian packages
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 6 Jun 2017 12:22:39 +0200
+
 ceph (12.0.3-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Tue, 16 May 2017 12:42:53 +0000
 
+ceph (12.0.2-pve1) stable; urgency=medium
+
+  * rebase on top of v12.0.2
+
+  * cherry-pick monitor upgrade fix for upgrades from <= v12.0.2
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 26 Apr 2017 10:20:20 +0200
+
 ceph (12.0.2-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 20 Apr 2017 19:59:57 +0000
 
+ceph (12.0.1-pve1) stable; urgency=medium
+
+  * revert dropping of boost build dependencies
+
+  * revert including our package release in ceph version output
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 29 Mar 2017 09:43:26 +0200
+
 ceph (12.0.1-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Fri, 24 Mar 2017 15:47:57 +0000
 
+ceph (12.0.0-pve3) stable; urgency=medium
+
+  * make ceph version output even more explicit
+
+  * enable systemd targets by default
+
+  * fix IPv6 monitor creation bug
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 22 Mar 2017 09:02:37 +0100
+
+ceph (12.0.0-pve2) stable; urgency=medium
+
+  * support Debian Stretch in ceph-detect-init
+
+  * fix "ceph version" output
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 21 Mar 2017 09:58:27 +0100
+
+ceph (12.0.0-pve1) stable; urgency=medium
+
+  * Rebuild for PVE 5.0 / Debian Stretch
+
+ -- Proxmox Support Team <support@proxmox.com>  Mon, 20 Mar 2017 09:12:32 +0100
+
 ceph (12.0.0-1) stable; urgency=medium
 
   * New upstream release
-- 
2.1.4

