From a035a93901ca31fb65c631058c204571237b2ac2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Mon, 20 Mar 2017 09:12:50 +0100
Subject: [PATCH 1/3] update changelog
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/changelog | 108 +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 108 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 3fef34d2b7..2c92d9c4ab 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,57 +1,165 @@
+ceph (12.2.0-pve1) stable; urgency=medium
+
+  * rebase on latest upstream release
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 30 Aug 2017 08:26:23 +0200
+
 ceph (12.2.0-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Mon, 28 Aug 2017 16:30:16 +0000
 
+ceph (12.1.4-pve1) unstable; urgency=medium
+
+  * rebase on latest upstream Release Candidate
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 16 Aug 2017 08:02:47 +0200
+
 ceph (12.1.4-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Tue, 15 Aug 2017 13:45:08 +0000
 
+ceph (12.1.3-pve1) unstable; urgency=medium
+
+  * rebase on latest upstream Release Candidate
+
+ -- Proxmox Support Team <support@proxmox.com>  Mon, 14 Aug 2017 08:44:52 +0200
+
 ceph (12.1.3-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 10 Aug 2017 19:22:38 +0000
 
+ceph (12.1.2-pve1) unstable; urgency=medium
+
+  * rebase on latest upstream Release Candidate
+
+ -- Proxmox Support Team <support@proxmox.com>  Thu, 3 Aug 2017 11:57:13 +0200
+
 ceph (12.1.2-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Tue, 01 Aug 2017 17:55:37 +0000
 
+ceph (12.1.1-pve1) unstable; urgency=medium
+
+  * rebase on latest upstream Release Candidate
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 19 Jul 2017 11:22:27 +0200
+
 ceph (12.1.1-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Mon, 17 Jul 2017 16:55:59 +0000
 
+ceph (12.1.0-pve2) unstable; urgency=medium
+
+  * fix https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=582819
+
+  * add 0004-debian-workaround-the-bug-in-dpkg-maintscript-helper.patch
+
+ -- Proxmox Support Team <support@proxmox.com>  Mon, 03 Jul 2017 14:03:31 +0200
+
+ceph (12.1.0-pve1) stable; urgency=medium
+
+  * rebase on latest upstream Release Candidate
+
+  * enable new restful mgr module by default
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 27 Jun 2017 09:44:46 +0200
+
 ceph (12.1.0-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 22 Jun 2017 15:43:47 +0000
 
+ceph (12.0.3-pve3) stable; urgency=medium
+
+  * update logrotate fix to final version
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 20 Jun 2017 09:27:53 +0200
+
+ceph (12.0.3-pve2) stable; urgency=medium
+
+  * fix Ceph #19938: two logrotate scripts when upgrading from Debian packages
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 7 Jun 2017 10:20:06 +0200
+
+ceph (12.0.3-pve1) stable; urgency=medium
+
+  * rebase on top of v12.0.3
+
+  * don't enable (broken) ceph-mgr module by default
+
+  * use bundled boost
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 6 Jun 2017 12:22:39 +0200
+
 ceph (12.0.3-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Tue, 16 May 2017 12:42:53 +0000
 
+ceph (12.0.2-pve1) stable; urgency=medium
+
+  * rebase on top of v12.0.2
+
+  * cherry-pick monitor upgrade fix for upgrades from <= v12.0.2
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 26 Apr 2017 10:20:20 +0200
+
 ceph (12.0.2-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 20 Apr 2017 19:59:57 +0000
 
+ceph (12.0.1-pve1) stable; urgency=medium
+
+  * revert dropping of boost build dependencies
+
+  * revert including our package release in ceph version output
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 29 Mar 2017 09:43:26 +0200
+
 ceph (12.0.1-1) stable; urgency=medium
 
   * New upstream release
 
  -- Ceph Release Team <ceph-maintainers@ceph.com>  Fri, 24 Mar 2017 15:47:57 +0000
 
+ceph (12.0.0-pve3) stable; urgency=medium
+
+  * make ceph version output even more explicit
+
+  * enable systemd targets by default
+
+  * fix IPv6 monitor creation bug
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 22 Mar 2017 09:02:37 +0100
+
+ceph (12.0.0-pve2) stable; urgency=medium
+
+  * support Debian Stretch in ceph-detect-init
+
+  * fix "ceph version" output
+
+ -- Proxmox Support Team <support@proxmox.com>  Tue, 21 Mar 2017 09:58:27 +0100
+
+ceph (12.0.0-pve1) stable; urgency=medium
+
+  * Rebuild for PVE 5.0 / Debian Stretch
+
+ -- Proxmox Support Team <support@proxmox.com>  Mon, 20 Mar 2017 09:12:32 +0100
+
 ceph (12.0.0-1) stable; urgency=medium
 
   * New upstream release
-- 
2.11.0

