From 653efebdc16c1263d5835c5ae972f95a8ecf97f4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Wed, 22 Mar 2017 09:00:34 +0100
Subject: [PATCH 3/3] enable systemd targets by default
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index 2e4ec9d22a..64278d79d1 100755
--- a/debian/rules
+++ b/debian/rules
@@ -130,6 +130,9 @@ override_dh_installinit:
 
 	dh_installinit -p ceph-base --name ceph --no-start
 	dh_installinit -p radosgw --no-start
+	dh_systemd_enable -p ceph-common ceph.target
+	dh_systemd_enable -p ceph-mon ceph-mon.target
+	dh_systemd_enable -p ceph-osd ceph-osd.target
 
 	# NOTE: execute systemd helpers so they pickup dh_install'ed units and targets
 	dh_systemd_enable
-- 
2.11.0

