From cf2641af1f3e3c86e7a6cfdadd8967b837b88893 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Wed, 22 Mar 2017 09:00:34 +0100
Subject: [PATCH 4/6] enable systemd targets by default
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index 3810602..9c1b33c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -124,6 +124,9 @@ override_dh_installinit:
 
 	dh_installinit -p ceph-base --name ceph --no-start
 	dh_installinit -p radosgw --no-start
+	dh_systemd_enable -p ceph-common ceph.target
+	dh_systemd_enable -p ceph-mon ceph-mon.target
+	dh_systemd_enable -p ceph-osd ceph-osd.target
 
 override_dh_systemd_start:
 	dh_systemd_start --no-restart-on-upgrade
-- 
2.1.4

