From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alwin Antreich <a.antreich@proxmox.com>
Date: Wed, 24 Oct 2018 10:38:19 +0200
Subject: [PATCH] debian/control: add break libpvestorage-perl

---
 debian/control | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/control b/debian/control
index 83dc414788..1cfa2250de 100644
--- a/debian/control
+++ b/debian/control
@@ -153,6 +153,7 @@ Replaces: ceph (<< 10),
 Breaks: ceph (<< 10),
         ceph-test (<< 12.2.2-14),
         python-ceph (<< 0.92-1223),
+        libpve-storage-perl (<= 5.0-25),
 Description: common ceph daemon libraries and management tools
  Ceph is a massively scalable, open-source, distributed
  storage system that runs on commodity hardware and delivers object,
@@ -634,6 +634,7 @@ Breaks: ceph (<< 10),
         ceph-test (<< 9.0.3-1646),
         librbd1 (<< 0.92-1238),
         python-ceph (<< 0.92-1223),
+        libpve-storage-perl (<= 5.0-25),
 	radosgw (<< 12.0.3)
 Suggests: ceph-base (= ${binary:Version}),
           ceph-mds (= ${binary:Version}),
