From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Tue, 2 Jun 2020 17:20:44 +0200
Subject: [PATCH] d/control: move python dependencies over to python3

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 debian/control | 24 ++++++++++++------------
 debian/rules   | 10 ------------
 2 files changed, 12 insertions(+), 22 deletions(-)

diff --git a/debian/control b/debian/control
index c93aa3c21..aa8edb3d2 100644
--- a/debian/control
+++ b/debian/control
@@ -138,7 +138,7 @@ Depends: binutils,
          psmisc,
          xfsprogs,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Recommends: btrfs-tools,
             ceph-mds (= ${binary:Version}),
@@ -184,7 +184,7 @@ Package: cephadm
 Architecture: linux-any
 Recommends: podman (>= 2.0.2) | docker.io
 Depends: lvm2,
-	 ${python:Depends},
+	 ${python3:Depends},
 Description: cephadm utility to bootstrap ceph daemons with systemd and containers
  Ceph is a massively scalable, open-source, distributed
  storage system that runs on commodity hardware and delivers object,
@@ -237,7 +237,8 @@ Depends: ceph-base (= ${binary:Version}),
          python3-requests,
          python3-werkzeug,
+         python3-yaml,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Suggests: ceph-mgr-dashboard,
           ceph-mgr-diskprediction-local,
@@ -263,7 +263,7 @@ Depends: ceph-mgr (= ${binary:Version}),
          python3-werkzeug,
          python3-routes,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Description: dashboard module for ceph-mgr
  Ceph is a massively scalable, open-source, distributed
@@ -284,7 +284,7 @@ Depends: ceph-mgr (= ${binary:Version}),
          python3-scipy,
          python3-sklearn,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Description: diskprediction-local module for ceph-mgr
  Ceph is a massively scalable, open-source, distributed
@@ -297,7 +297,7 @@ Description: diskprediction-local module for ceph-mgr
 Package: ceph-mgr-modules-core
 Architecture: all
 Depends: ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
          python3-dateutil,
          python3-openssl,
@@ -317,7 +317,7 @@ Depends: ceph-mgr (= ${binary:Version}),
 Depends: ceph-mgr (= ${binary:Version}),
          python3-jsonpatch
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Description: rook module for ceph-mgr
  Ceph is a massively scalable, open-source, distributed
@@ -333,7 +333,7 @@ Architecture: all
 Depends: ceph-mgr (= ${binary:Version}),
          python3-kubernetes,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
 Description: kubernetes events module for ceph-mgr
  Ceph is a massively scalable, open-source, distributed
  storage system that runs on commodity hardware and delivers object,
@@ -348,7 +348,7 @@ Depends: ceph-mgr (= ${binary:Version}),
 Depends: ceph-mgr (= ${binary:Version}),
 	 cephadm,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          openssh-client,
          python3-jinja2
 Description: cephadm orchestrator module for ceph-mgr
@@ -412,7 +412,7 @@ Depends: ceph-base (= ${binary:Version}),
          lvm2,
 	 sudo,
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Replaces: ceph (<< 10), ceph-test (<< 12.2.2-14)
 Breaks: ceph (<< 10), ceph-test (<< 12.2.2-14)
@@ -620,7 +620,7 @@ Depends: librbd1 (= ${binary:Version}),
          python3-rbd (= ${binary:Version}),
          python3-rgw (= ${binary:Version}),
          ${misc:Depends},
-         ${python:Depends},
+         ${python3:Depends},
          ${shlibs:Depends},
 Conflicts: ceph-client-tools,
 Replaces: ceph (<< 10),
diff --git a/debian/rules b/debian/rules
index c93aa3c21..aa8edb3d2 100755
--- a/debian/rules
+++ b/debian/rules
@@ -125,16 +125,6 @@
 override_dh_shlibdeps:
 	dh_shlibdeps -a --exclude=erasure-code --exclude=rados-classes --exclude=compressor
 
-override_dh_python3:
-	for binding in rados cephfs rbd rgw; do \
-	  dh_python3 -p python3-$$binding;      \
-	done
-	dh_python3 -p python3-ceph-argparse
-	dh_python3 -p python3-ceph-common
-	dh_python3 -p cephfs-shell
-	dh_python3 -p cephfs-top
-	dh_python3 -p cephadm
-
 # do not run tests
 override_dh_auto_test:
 
