From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Fri, 15 Nov 2019 15:22:30 +0100
Subject: [PATCH] bump version to 14.2.4.1

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4e187f261b..907db98a96 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.5.1)
 
 project(ceph CXX C ASM)
-set(VERSION 14.2.4)
+set(VERSION 14.2.4.1)
 
 if(POLICY CMP0028)
   cmake_policy(SET CMP0028 NEW)
