From 1f4b106d49fc916994d97e273599f75caa904c3b Mon Sep 17 00:00:00 2001
From: Mark Nelson <mark.nelson@clyso.com>
Date: Thu, 14 Dec 2023 05:19:46 +0000
Subject: [PATCH] debian/rules: Fix build_type for massive performance gain

Signed-off-by: Mark Nelson <mark.nelson@clyso.com>
---
 debian/rules | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules b/debian/rules
index ed7f4a255ed4b..b28abb7d62788 100755
--- a/debian/rules
+++ b/debian/rules
@@ -29,6 +29,7 @@ extraopts += -DWITH_PYTHON3=3
 extraopts += -DWITH_CEPHFS_JAVA=ON
 extraopts += -DWITH_CEPHFS_SHELL=ON
 extraopts += -DWITH_SYSTEMD=ON -DCEPH_SYSTEMD_ENV_DIR=/etc/default
+extraopts += -DCMAKE_BUILD_TYPE=RelWithDebInfo
 extraopts += -DWITH_GRAFANA=ON
 ifeq ($(DEB_HOST_ARCH), amd64)
   extraopts += -DWITH_RBD_RWL=ON
